client/react/frontpage/components/BlogPostInfo.js

```diff
@@ -42,6 +44,9 @@ const BlogPostInfo = React.createClass({
     return null;
   },
   render() {
+    if (this.state.fetching) {
+      return <p />;
```

Comment:
So we decided on not showing this feature altogether if it hasn't finished loading yet?

Reply:
yeah, before it was just showing a default photo and empty author while fetching so i thought that this would be a better solution. it usually only takes a few seconds to load